.vscode/launch.json

```diff
@@ -14,6 +14,20 @@
             "run-script", "dev"
         ],
         "port": 5858
+      },
+      {
+        "name": "Mocha current file",
+        "type": "node",
+        "request": "launch",
+        "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+        "args": [
+          "--timeout", "999999",
+          "--colors",
+          "${file}",
+          "--require", "@babel/register",
+        ],
+        "console": "integratedTerminal",
+        "internalConsoleOptions": "neverOpen"
```

Comment:
This is unrelated to this PR. Just added VSCode build so we can debug mocha tests.